body.tex

```diff
@@ -1,215 +1,336 @@
+in the corresponding LSST the Docs landing page. Contributors, reviewers and stakeholders can easily access the produced pdf,
+check the content of the Test Plan, and comment, ask for clarification, or request changes using the GitHub
 Pull Request (PR) mechanism or the corresponding Jira issue.
-When agreement on the test plan content has been reached, the ATM test plan status is changed to \textbf{approved}. The test activity is ready to start.
+When agreement has been reached, the TM4J Test Plan status is changed to \textbf{Approved}. The test activity is ready to start.
 
-The outcome of this first phase is an approved test plan to be uploaded in Docushare. 
+The outcome of this first phase is an approved \textit{Test Plan and Report} document to be uploaded in Docushare.
 The document at this stage provides the agreed test procedures and all the necessary information required to execute the tests.
-The Github repository will be tagged with the corresponding issue number \textit{v1.0}.
-
-\paragraph{Execution}:
-In this phase, the testers identified in the the test plan are in charge to execute the test procedures and 
-document the result of each steps in Jira ATM.
-The ATM plugin provide a test player view, where for each step in each test case, it is possible to say, it it has been executed successfully or not.
-Also it possible to related to the test execution any Jira issue documenting problems raised during the execution.
 
-All this information is extracted in a report document.
-In order to avoid the proliferation of documents, the  report information is added to the \textit{Test Plan} created in the previous fare.
-The document is therefore renamed as \textbf{Test Plan and Report}.
+\paragraph{Execution}
+In this phase, the testers, identified in the previous phase, are  executing the test procedures and
+documenting the result of each steps in Jira.
+The TM4J plugin provides a test player view, where for each step in each test case, it is possible to say if it has been executed successfully or not,
+specify the result of the step, and relate any Jira issue that has been found during the test execution.
 
-Once the test execution is completed, an overall assessment shall be provided in the ATM Test Plan.
+The test result information is extracted and added to the \textit{Test Plan and Report} document.
+Once the test execution is completed, an overall assessment must  be provided in the TM4J Test Plan.
 
-The document generation will provide a readable version of the document including all test information.
-Stakeholder are able to to review the outcome of the test campaign using the same PR mechanism reported above, 
+As done in the previous phase, the document is generated automatically from Jira and provides an easy way to access the test campaign information.
+Stakeholders can review the outcome of the test campaign using the same PR mechanism reported above,
```

Comment:
So here,  I think the role of the Product Owner needs to be introduced. It is the PO who must finally approve and sign off on the test report (and also on the test plan above prior to starting the testing).

Reply:
Rephasing. But this will require a second round.